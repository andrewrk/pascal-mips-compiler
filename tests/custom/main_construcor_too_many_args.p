program A;
class A begin
    function A(i : integer); begin
        i := i
    end
end
.
