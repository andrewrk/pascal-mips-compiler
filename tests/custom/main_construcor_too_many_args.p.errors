ERROR: line 3: constructor for main class "A" must have no parameters
