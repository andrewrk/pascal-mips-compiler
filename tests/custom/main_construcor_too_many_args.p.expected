ERROR: line 2: constructor for main class "A" must have no parameters
